src/js/components/organisms/hero-product/hero-product.ts

```diff
@@ -15,7 +15,7 @@ export default class HeroProduct extends Vue implements HeroProductProps {
     @Prop({ type: Object, required: false }) image?: HeroProductProps['image'] | undefined;
     @Prop({ type: Object, required: false }) ctas?: HeroProductProps['ctas'] | undefined;
     @Prop({ type: Array, required: true }) products!: HeroProductProps['products'];
-    @Prop({ type: Array, required: true }) linkList!: HeroProductProps['linkList'];
+    @Prop({ type: Array, required: false }) linkList?: HeroProductProps['linkList'];
```

Comment:
You still need to add a v-if to the template to prevent it from iteration over a non existing object:

```
<ul class="hero__linklist" >
<template v-for="link in linkList">
'''
Should be:
```
<ul class="hero__linklist"  v-if="linkList">
<template v-for="link in linkList">
```